### DISCUSSION TITLE
How to do hyphening

### DISCUSSION THREAD

**Discussion by powermetaldev** [LibrePDF/OpenPDF#1205] [Q&A]
Hey community. 
I want to use the hyphening functionality in OpenPDF (Phrase withing PdfPCell) but there is no documentation on how to do it.
Just creating an HyphenationAuto object (for german) and setting it via setHyphenation doesn't work as it can't find the "config"-file.

Has anybody of you any Idea about what to do exactly?

When dependencies are needed, it'd be awesome if you can provide the maven deps :-) 

Thx in advance

**Comment by arnthom** [comment]
Hi skoesi!

Comparing my code with your description shows no obvious error.
Do you import "itext-hyph-xml.jar"?
